app/scripts/views/delete_account.js

```diff
@@ -27,6 +30,14 @@ function (Cocktail, BaseView, FormView, Template, Session, AuthErrors,
     template: Template,
     className: 'delete-account',
 
+    events: {
```

Comment:
The DOM events declarations and `_openSettingsUnit` are repeated in each module with the `Settings*` mixins. I'd move the `events` declaration to the mixin, which has its own `_openSettingsUnit` function. The `_openSettingsUnit` function would pull the URL to open from either an href or a data attribute of the `.settings-unit-toggle` element.